dotnetcore3.1/cookiecutter-aws-sam-quick-start-cloudwatch-events-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "",
+  "region": "",

```

Comment:
nit: Can we remove empty fields from this file?

Reply:
@mndeveci Done!